### ISSUE TITLE
Training Hyper-Parameters

### ISSUE THREAD

**Issue by bowen-upenn** [Yuqifan1117/CaCao#24]
Thank you for your amazing work! We are trying to replicate your results. The paper mentions 

> "CaCao can generate extra training data D automatically in a low-resource way and flexibly extend the current dataset by fine-grained predicate boosting. Finally, we retrain the refined SGG models with enhanced data for a more balanced prediction." 

Does it mean you simply run the training script from [Scene-Graph-Benchmark](https://github.com/KaihuaTang/Scene-Graph-Benchmark.pytorch/tree/master) but with your expanded dataset `VG-SGG-base-EXPANDED-with-attri.h5` and `VG-SGG-base-EXPANDED-dicts-with-attri.json` in the final step of the model retraining (closed-world)? 

If so, may we ask the training hyper-parameters you use, such as SOLVER.BASE_LR and SOLVER.MAX_ITER, to train the motif/vctree/transformer baseline models with the expanded dataset `VG-SGG-base-EXPANDED-with-attri.h5` and `VG-SGG-base-EXPANDED-dicts-with-attri.json`? Thank you in advance for your help!

**Comment by Yuqifan1117** [comment]
Thanks for your attention! If correctly, we seem to run the training script in  [FGPL](https://github.com/XinyuLyu/FGPL) for motif/vctree/transformer baseline model to take full advantage of the extended tail data from CaCao.

Here are our training hyper-parameters for motifs (others are similar to original settings):
'MODEL.ROI_RELATION_HEAD.PREDICTOR', 'MotifPredictor', 'MODEL.ROI_RELATION_HEAD.PREDICT_USE_BIAS', 'True', 'DTYPE', 'float32', 'SOLVER.IMS_PER_BATCH', '16', 'TEST.IMS_PER_BATCH', '1', 'SOLVER.MAX_ITER', '16000', 'SOLVER.BASE_LR', '1e-3', 'SOLVER.SCHEDULE.TYPE', 'WarmupMultiStepLR', 'SOLVER.STEPS', '(10000, 16000)', 'SOLVER.VAL_PERIOD', '2000', 'SOLVER.CHECKPOINT_PERIOD', '16000',
